cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -336,7 +355,7 @@ private void removeThrottledReplicasFromTopic(String topic, Set<String> replicas
     }
   }
 
-  private void removeThrottledRateFromBroker(Integer brokerId)
+  private void removeThrottledRatesFromBroker(Integer brokerId)
```

Comment:
Is this method now become "remove both throttle rates"? In that case we should add a comment to specify that.

Reply:
Oh no, it is just a simple mistake. There are no changes in the method body. fixed.